core/common/src/test/java/alluxio/UnderFileSystemFactoryRegistryRule.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import alluxio.underfs.UnderFileSystemFactory;
+import alluxio.underfs.UnderFileSystemFactoryRegistry;
+
+public class UnderFileSystemFactoryRegistryRule extends AbstractResourceRule {
+  private UnderFileSystemFactory mFactory = null;
```

Comment:
nit: initialize a member variable to `null` is unnecessary as `null` is the default value